jwst/resample/resample.py

```diff
@@ -500,22 +487,23 @@ def resample_many_to_one(self, input_models):
         if driz.out_ctx is not None:
             output_model.con = driz.out_ctx
 
-        del driz
-
-        # compute final variances:
-        self._compute_resample_variance_totals(output_model)
-
-        var_components = [
-            output_model.var_rnoise,
-            output_model.var_poisson,
-            output_model.var_flat
-        ]
-        output_model.err = np.sqrt(np.nansum(var_components, axis=0))
-
-        # nansum returns zero for input that is all NaN -
-        # set those values to NaN instead
-        all_nan = np.all(np.isnan(var_components), axis=0)
-        output_model.err[all_nan] = np.nan
+        valid_var = []
+        for k, name in enumerate(var_list):
+            if invalid_var[k]:
+                var = np.full_like(output_model.data, np.nan)
+            else:
+                var = driz.out_img2[k]
+                valid_var.append(var)
+            setattr(output_model, name, var)
+
+        if len(valid_var) > 0:
+            all_nan = np.all(np.isnan(valid_var), axis=0)
+            err = np.nansum(valid_var, axis=0).astype(np.float32)
+            err[all_nan] = np.nan
+        else:
+            err = np.full_like(output_model.data, np.nan)
+        output_model.err = err
```

Comment:
Closer, thank you!  But I think we still need to take the square root for the error.

Reply:
Oh, I forgot